From d6e96880c8c82898296b592e6fe4bb291995f5aa Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 25 Jan 2025 19:41:10 +0800
Subject: [PATCH 6/7] FEDORA: Convert to the expected pointer

This was not fixed before the file was removed upstream.

Link: https://src.fedoraproject.org/rpms/gtk2/blob/a3a398f4f95684dba7a461acf389626eb2a3b90e/f/gtk2-c89-6.patch
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 modules/other/gail/gailtreeview.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/other/gail/gailtreeview.c b/modules/other/gail/gailtreeview.c
index 23923c59a1..ac6ff0a1c4 100644
--- a/modules/other/gail/gailtreeview.c
+++ b/modules/other/gail/gailtreeview.c
@@ -995,7 +995,7 @@ gail_tree_view_ref_child (AtkObject *obj,
         gail_cell_add_state (cell, ATK_STATE_FOCUSABLE, FALSE);
         if (focus_index == i)
           {
-            gailview->focus_cell = g_object_ref (cell);
+            gailview->focus_cell = ATK_OBJECT (g_object_ref (cell));
             gail_cell_add_state (cell, ATK_STATE_FOCUSED, FALSE);
             g_signal_emit_by_name (gailview,
                                    "active-descendant-changed",
-- 
2.48.1

